package org.davilo.app.di.network.utils

class MyException constructor(val myMessage: String) : Throwable()
